README.md

```diff
@@ -24,7 +24,7 @@ technologies and release cycles.
 It combines **asynchronous programming principles** and **message-driven architecture** providing **a scalable
 platform** for modern sites.
 
-**Knot.x** connects all of the above **in a unified customer experience**.
+**Knot.x** connects all of the above **into a unified customer experience**.
```

Comment:
I don't think you can _connect something **into** something_. Both the old and the new version sound strange. How about `**Knot.x** connects all of the above **to create a unified customer experience**.`